cubic-server/Server.cpp

```diff
@@ -165,32 +156,20 @@ void Server::launch(const configuration::ConfigHandler &config)
 
     _writeThread = std::thread(&Server::_writeLoop, this);
 
+#if PROMETHEUS_SUPPORT == 1
+    if (CONFIG["monitoring-prometheus-enable"].as<bool>()) {
+        _prometheusExporter = std::make_unique<PrometheusExporter>("0.0.0.0:4242");
```

Comment:
Is this really the default port for prometheus ? :')

Reply:
Can it be 8080?

Reply:
Bruh I forgot to make it a config xd

Reply:
Lemme fix that real quick